overlord/snapstate/handlers.go

```diff
@@ -792,31 +792,13 @@ func (m *SnapManager) undoMountSnap(t *state.Task, _ *tomb.Tomb) error {
 	return m.backend.RemoveSnapDir(snapsup.placeInfo(), otherInstances)
 }
 
-// queryDisabledServices returns a list of all currently disabled snap services
-// in the snap.
+// queryDisabledServices uses wrappers.QueryDisabledServices()
+//
 // Note this function takes a snap info rather than snapst because there are
 // situations where we want to call this on non-current snap infos, i.e. in the
 // undo handlers, see undoLinkSnap for an example.
 func (m *SnapManager) queryDisabledServices(info *snap.Info, pb progress.Meter) ([]string, error) {
```

Comment:
this level of wrapping can go away already here, no? am I missing something?  this functions is used in two places and handlers do `m.backend.Something` all the time

Reply:
It's there for tests currently, I can look into this if it can be simplified.

Reply:
@stolowski any preference here, do you prefer to fix this in your work?